bfh/fields.py

```diff
@@ -90,6 +94,18 @@ def field_name(self):
     def field_name(self, value):
         self._field_name = value
 
+    @property
+    def default(self):
+        return getattr(self, "_default", None)
+
+    @default.setter
+    def default(self, value):
+        if callable(value):
+            default = value()
```

Comment:
By evaluating in the setter you lose the point of making this callable. A mutable default still has the same problem as it did if you set it plain:

```python
In [28]: class Wow(bfh.Schema):
    foo = bfh.fields.UnicodeField(default={"wow": 1})
   ....
In [29]: my1 = Wow(foo=None)

In [30]: my2 = Wow(foo=None)

In [31]: oops = my1.foo

In [32]: oops['wow'] = 2

In [33]: my2.foo
Out[39]: {'wow': 2}
```